### ISSUE TITLE
Cannot use import outside a module

### ISSUE THREAD

**Issue by JasterV** [benawad/tsconfig.json#40]
For some reason this tsconfig does not allow es modules:

```
(node:4907) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
Waiting for the debugger to disconnect...
/home/jasterv/crud/dist/index.js:1
import app from './api/server';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1025:15)
    at Module._compile (node:internal/modules/cjs/loader:1059:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47
```

**Comment by timthedev07** [comment]
What type of project are you building? Or, what are you using typescript for?

**Comment by sunjc826** [comment]
Using `npx tsconfig.json` and selecting `node` (the 2nd option on my terminal) doesn't give the right file. Here is the file that I got. (attached below). Most notably @JasterV, the reason why this version doesn't work is that the compilerOptions.module says `es2020` instead of `commonjs`, so imports in your ts files aren't being compiled to require. So a fix is to just copy the file directly from [https://github.com/benawad/tsconfig.json/blob/master/src/config/tsconfig.node(Default).json](https://github.com/benawad/tsconfig.json/blob/master/src/config/tsconfig.node(Default).json).
I tested `npx tsconfig.json` with the other file types, e.g. react, ts-cli etc and compared them with the files in this repository, and they matched as far as I can see.
[tsconfig.json.txt](https://github.com/benawad/tsconfig.json/files/6873708/tsconfig.json.txt)
